support-frontend/assets/helpers/redux/selectors/formValidation/personalDetailsValidation.ts

```diff
@@ -40,3 +40,16 @@ export function getPersonalDetailsErrors(
 		...stateOrProvinceErrors,
 	};
 }
+
+export function getUserCanTakeOutContribution(
```

Comment:
would be good to call this `getUserCanTakeOutRecurringContribution` because they are still able to take out a single contribution

Reply:
The function accounts for this by always returning `true` for one-off contributions.